src/components/timeline/TrackContextMenu.js

```diff
@@ -516,13 +523,29 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     );
   }
 
+  renderShowAllTracks() {
+    const { rightClickedTrack } = this.props;
+    if (rightClickedTrack !== null) {
+      return null;
+    }
+    return (
+      <React.Fragment>
+        <MenuItem onClick={this._showAllTracks}>
```

Comment:
Suggestion: what do you think of making this item "disabled" if all tracks are already visible? I believe you can check this with something like:
```js
const hiddenLocalTracksCount = [...hiddenLocalTracksByPid.values()].reduce((total, set) => total + set.size, 0);
const shouldBeDisabled = (hiddenGlobalTracks.size + hiddenLocalTracksCount) === 0;
```